{{cookiecutter.project_slug}}/apps/pages/tests/test_blocks.py

```diff
@@ -0,0 +1,116 @@
+from django.test import SimpleTestCase
+
+from pages import blocks
+
+
+class TestBlockRendering(SimpleTestCase):
```

Comment:
Yay! ❤️ 
Block tests!

I'd be tempted to put each block in it's own class, so that different configurations / options are easier to add (as in churchinwales and internationallandcoalition) but I'm just happy to see them at all in the template!

Thanks 👍